Dockerfile.dev

```diff
@@ -0,0 +1,24 @@
+# Used by quay.io to trigger developer builds
+FROM quay.io/redhat-cne/openshift-origin-release:golang-1.17 AS builder
```

Comment:
I am not sure if this is okay , do we have a some kind of process  to follow  openshift release and update this ?

Reply:
I guess it is okay, just wanted to make sure this is not used downstream. because brew doesn't have access to pulling from external registries

Reply:
so from a downstream builds perspective I think we are going to continue using Dockerfile and this Dockerfile.dev is used for development and upstream testing. Is that correct @jzding ?

Reply:
yes. I expect Dockerfile.dev is used only for developer tests. I put the comment at the the first line that it is for `developer builds`. Downstream should use Dockerfile and official `origin-` images under quay.io/openshift.